libs/ui-toolkit/info-box/src/lib/info-box.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="info-box-container" [ngClass]="_class">
+  <ng-container *ngFor="let info of _infoList">
+    <div class="info-div">
+      <div *ngIf="info.title" class="info-title">{{info.title}}</div>
+      <div *ngIf="info.value" class="info-value">
+        <mur-arrow-variation *ngIf="!!info.arrow" size="lg" [value]="info.value" class="info-arrow"></mur-arrow-variation>
+        <gro-currency-display [ngClass]="{'report-smaller': !!info.smallerValue}"
+        [value]="info.value" [currencyCode]="'BRL'"
+        [symbolDisplay]="'symbol'" [digitInfo]="'1.2-2'"
+        [currencyClass]="'currency'" [hideNegative]="!!info.arrow"
+        ></gro-currency-display>
+      </div>
+      <div *ngIf="info.volatility && info.sharpe" class="info-vol-sharpe">
```

Comment:
The original template checked for the existence of the value rather than the value itself because zeroed values should still be displayed. Please verify that this component is still behaving as intended